### PR TITLE
Added type and name attribute to the inputArea driver + the necessary…

### PR DESCRIPTION
**Pull request by galewix**
… tests

### What changed
passed the element type and name attribute to the enterText driver function.
...

### Why it changed
While testing and using the enterText testKit method the name and type aren't being passed to the onChange handler in the event.
...
Please review this changes and if its fine by you I would like to add the name and type to all of the different input components.
---

- [x] Change is tested
- [x] Change is documented
- [ ] Build is green
- [ ] Change of UX is approved by [wuwa](https://github.com/wuwa) or [milkyfruit](https://github.com/milkyfruit)

### CODE REVIEW

File: src/InputArea/README.TESTKIT.md [hunk 1]
```diff
@@ -11,6 +11,8 @@
 | focus | - | - | focus the element |
 | enterText | string | - | applied text to the element |
 | getValue | - | string | get the element default text |
+| getName | - | string | get the element default text |
```
Comment (hepiyellow) [inline src/InputArea/README.TESTKIT.md]: Fix description: "get the native `textarea`'s `name` attribute`"

File: src/InputArea/README.TESTKIT.md [hunk 2]
```diff
@@ -11,6 +11,8 @@
...
+| getType | - | string | get the element default text |
```
Comment (hepiyellow) [inline src/InputArea/README.TESTKIT.md]: Fix description: "get the native `textarea`'s `type` attribute`"

File: src/InputArea/InputArea.js [hunk 3]
```diff
@@ -83,6 +85,8 @@ class InputArea extends WixComponent {
       <div className={styles.wrapper}>
         <div className={classes}>
           <textarea
+            type={type}
```
Comment (hepiyellow) [inline src/InputArea/InputArea.js]: You can not add `type` attribute to `textarea` (it is ignored)
Why do you need it?

`textarea`'s type attribute always returns `"textarea"`.
See [this fiddle](https://jsfiddle.net/vgpke5dt/)

File: src/InputArea/InputArea.spec.js [hunk 4]
```diff
@@ -26,6 +26,28 @@ describe('InputArea', () => {
     });
   });
 
+  describe('name attribute', () => {
```
Comment (hepiyellow) [inline src/InputArea/InputArea.spec.js]: Please add test that uses the `enterText` driver method and checks that it gets the proper `name`.

File: src/InputArea/README.TESTKIT.md (hunk 2 above)
Comment (galewix) [reply src/InputArea/README.TESTKIT.md]: thanks i will

File: src/InputArea/InputArea.js (hunk 3 above)
Comment (galewix) [reply src/InputArea/InputArea.js]: np you are right I'm afk ill make the changes later today
@hepiyellow

**Comment by hepiyellow** [comment]
@galewix Please next time run `npm run lint` and `npm run build` and `npm run test:unit`
before pushing.

**Comment by galewix** [comment]
> @galewix Please next time run `npm run lint` and `npm run build` and `npm run test:unit`
> before pushing.

np i will next time

**Pull request merged**